guides/common/modules/ref_capsule-server-scalability-considerations.adoc

```diff
@@ -5,10 +5,7 @@
 
 The maximum number of {SmartProxyServer}s that {ProjectServer} can support has no fixed limit.
 ifndef::satellite[]
-The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer} with {EL} 8 or {EL} 7.
-endif::[]
-ifndef::satellite[]
-The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer} with {EL} 8.
+The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer} with {EL} 7.
```

Comment:
```suggestion
The tested limit is 17 {SmartProxyServer}s with 2 vCPUs on a {ProjectServer}.
```

Maybe just leave out `with {EL} N`?

Reply:
Sometimes it can be so simple ... Like that.